cmake/PythonFixUpTclTk.py

```diff
@@ -100,46 +101,80 @@ def get_linked_libraries(p: Path):
 
 if __name__ == "__main__":
 
-    if platform.system() != "Darwin":
+    if platform.system() != "Darwin" and platform.system() != 'Linux':
```

Comment:
Now that we are packaging up the tcl and tk stuff for Ubuntu, we need to abort if we are not on one of those (so .... Windows :upside_down_face: )